text/0000-system-receptionist.md

```diff
@@ -0,0 +1,255 @@
+- **Start Date:** 2022-04-15
+- **RFC PR:** (leave this empty)
+
+# Systems and receptionists
+
+## Summary
+
+Actor discovery is an important part of the actor model. This RFC proposes the following for the purpose of making it easier to find discoverable actors within a system using the **receptionist** pattern:
+
+- The creation of an actor system via `createSystem(...)` 
+- A system receptionist via `system.receptionist`
+- Receptionist actions:
+  - Registering an actor ref: `register(key)`
+  - Unregistering an actor ref: `unregister(key)`
+
+## Motivation
+
+https://doc.akka.io/docs/akka/current/typed/actor-discovery.html
+
+Naturally, actors form a hierarchical relationship, such that it is easiest for a parent actor to communicate with a child actor, and for a child actor to _sometimes_ communicate with the parent, if it has knowledge of it. However, these communication patterns are more difficult:
+
+- An actor communicating with a sibling actor
+- An actor communicating with any arbitrary actor
+- A system-level actor (such as a logging or notification actor) being made available to any actor
+- Retrieving non-stale data from another actor
+- Grouping actors
+- Communicating with groups of related actors
```

Comment:
Is "grouping actors" different from "communicating with groups of related actors"? should those be merged into a single item? if not - could we clarify what's the other use case for grouping actors?
```suggestion
- Communicating with groups of related actors
```